src/workers/postUpdated.ts

```diff
@@ -294,19 +300,25 @@ const updatePost = async ({
   }
 
   const title = data?.title || databasePost.title;
-  const updateBecameVisible =
-    content_type === PostType.Freeform
-      ? databasePost.visible
-      : !databasePost.visible && !!title?.length;
 
   data.id = databasePost.id;
   data.title = title;
-  data.visible = updateBecameVisible;
-  data.visibleAt = updateBecameVisible
-    ? databasePost.visibleAt ?? data.metadataChangedAt
-    : null;
```

Comment:
Removed it from `fixData` because we need to know if current post exist to properly deduce current visible status.